src/sage/combinat/posets/hasse_diagram.py

```diff
@@ -1160,25 +1158,29 @@ def coxeter_transformation(self, algorithm='cython'):
 
         EXAMPLES::
 
-            sage: P = posets.PentagonPoset()._hasse_diagram                             # optional - sage.modules
-            sage: M = P.coxeter_transformation(); M                                     # optional - sage.libs.flint sage.modules
+            sage: # needs sage.modules
```

Comment:
This line is redundant because of the next line

Reply:
Done.